package.json

```diff
@@ -241,7 +241,6 @@
         "@types/node": "^8.10.45",
         "@types/request": "2.0.7",
         "azure-arm-resource": "^3.0.0-preview",
-        "copy-webpack-plugin": "^4.6.0",
```

Comment:
This PR didn't change package-lock.json, which it should have. Also run `npm audit fix` to make sure it gets rid of any security alerts it can

Reply:
https://github.com/microsoft/vscode-azurevirtualmachines/pull/48